ingest/ingestHelpers.py

```diff
@@ -161,27 +164,18 @@ def get_authors(ds):
 # get_distributions: object -> [distributions] for objects such as: datasets, publications, ...
 def get_distributions(ds):
     distributions = []
-    distributionList = [faux for faux in ds.objects(DCO.hasDistribution) if has_type(faux, DCAT.Distribution)]
+    distributionList = [faux for faux in ds.objects(DCAT.distribution) if has_type(faux, DCODATA.Distribution)]
     for distribution in distributionList:
-        accessURL = str(list(distribution.objects(DCO.accessURL))[0])
+
+        accessURL = list(distribution.objects(DCAT.accessURL))[0]
+        accessURL = accessURL.identifier
```

Comment:
Does this throw an error if there is no accessURL on the distribution?  if accessURL is None accessURL.identifier may throw an exception.

Perhaps this, and the line below, should be updated such as:

``accessURL = accessURL.identifier if accessURL else None``

If this never throws an error, do you know why not?